rules/cross-platform/initial_access_zoom_meeting_with_no_passcode.toml

```diff
@@ -26,7 +26,7 @@ references = [
 risk_score = 47
 rule_id = "58ac2aa5-6718-427c-a845-5f3ac5af00ba"
 severity = "medium"
-tags = ["Elastic", "SecOps", "Continuous Monitoring", "Configuration Audit"]
+tags = ["Elastic", "Continuous Monitoring", "SecOps", "Configuration Audit"]
```

Comment:
```suggestion
tags = ["Elastic", "Application", "Communication", "Zoom", "Continuous Monitoring", "SecOps", "Configuration Audit"]
```

Reply:
This might be a bit verbose...

Reply:
I'd keep it as is for now since we have nothing else for `"Application", "Communication", "Zoom"` currently